_developments/git.md

```diff
@@ -8,11 +8,13 @@ weight: 1
 ## On Commits
 
 Try to keep each commit message concise, relevant, and _release-notes-ready_:
+
```

Comment:
unless they're needed, usually refrain from introducing unnecessary line changes as it could distract reviewers from focusing on the actual materials